WaterJug.py

```diff
@@ -4,101 +4,102 @@
 
 class WaterJug:
 
-    def __init__(self,fileptr):
-        self.num=int(fileptr.readline())
+    def __init__(self,fileptr): #constructor
+        self.num=int(fileptr.readline()) #number of glass
```

Comment:
Instead of having a comment here, if you can update variable name to be more explicit, that would be better.